evoMPS/mps_gen.py

```diff
@@ -307,7 +307,7 @@ def _correct_bond_dimension(self):
         self.D = correct_bond_dim_open_chain(self.D, self.q)
                 
     
-    def update(self, restore_CF=True, normalize=True, auto_truncate=False, restore_CF_after_trunc=True):
+    def update(self, restore_CF=True, normalize_CF = True, normalize=True, auto_truncate=False, restore_CF_after_trunc=True):
```

Comment:
I think we should use the existing `normalize` argument, rather than add a new `normalize_CF` argument.